main.tex

```diff
@@ -410,6 +410,49 @@
 % creates the second title. It will be ignored for other modes.
 \IEEEpeerreviewmaketitle
 
+\newpage	
+\section{Formal methods for EES}
+
+
+Systems reconfigurability is crucial for a variety of safety critical applications. 
+
+Formal methods - a mathematical model driven method provides a systematic approach
+for developing complex systems in a design reusable and correct-by-construction
+manner. 
+
+
+%A number of formal models exists for specification of process-based systems for
```

Comment:
No need to comment out text that is not final, let's uncomment everything so it's easier to judge the page limit.